tools/prom-metrics-collector/metrics_collector.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	parser "github.com/kubevirt/monitoring/pkg/metrics/parser"
+	"kubevirt.io/ssp-operator/internal/operands/metrics"
+
+	dto "github.com/prometheus/client_model/go"
+)
+
+var excludedMetrics = map[string]struct{}{
+	"kubevirt_ssp_operator_up_total":           {},
+	"kubevirt_ssp_template_validator_up_total": {},
+}
+
+func readMetrics() []*dto.MetricFamily {
+	var metricFamilies []*dto.MetricFamily
+	metrics := metrics.RecordRulesDescList
+
+	//metricsList := make([]parser.Metric, len(sspMetrics))
+	for _, metric := range metrics {
```

Comment:
change the name to sspMetric instead of metric

Reply:
Not sure it's needed, as we are in SSP repo.